sql/2024/fonts/development/styles_font_variant.sql

```diff
@@ -0,0 +1,71 @@
+-- Section: Development
+-- Question: Which features are used via font-variant in CSS?
+
+CREATE TEMPORARY FUNCTION PROPERTIES(json STRING)
```

Comment:
The previous version used to compare `font-variant` settings vs. `font-feature-settings`. The `font-feature-settings` data seems to be missing.

Reply:
Nevermind, I see that is now in `style_feature_control.sql`.